assets/js/components/Work/UIWorkCardItem.jsx

```diff
@@ -1,64 +1,78 @@
-import React from "react";
+import React, { useContext } from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
 import { setVisibilityClass, formatDate } from "../../services/helpers";
+import { IIIFProvider, IIIFContext } from "../../components/IIIF/IIIFProvider";
 
-const WorkCardItem = ({ work }) => {
+const WorkCardItem = ({
+  id,
+  representativeImage,
+  title,
+  descriptiveMetadata,
```

Comment:
Do we need `descriptiveMetadata`?   Looks like it's just used for it's `title` property?  Wonder how it's different from `title` in this object?   I'd say let's lose the larger (in the future) `descriptiveMetadata`, and keep it thin.

Reply:
`descriptiveMetadata` contains title and description fields of a work object. For search results, they are returned separately. For the actual work object in GraphiQL schema, `descriptiveMetadata` is an object with title, description and other fields.

Reply:
Yeah. `descriptiveMetadata` is not really a concept in the Elasticsearch documents